### PR TITLE
shipit_code_coverage: Show commit description in low coverage notifications

### LINKED ISSUE #851: shipit_code_coverage: Add commit message in the low coverage notifications

### PR DESCRIPTION
**Pull request by marco-c**
Fixes #851

### CODE REVIEW

File: src/shipit_code_coverage/shipit_code_coverage/notifier.py [hunk 1]
```diff
@@ -38,7 +38,9 @@ def notify(self):
         changesets = sum((data['changesets'] for data in push_data['pushes'].values()), [])
 
         for changeset in changesets:
-            if any(text in changeset['desc'] for text in ['r=merge', 'a=merge']):
+            desc = changeset['desc'].split('\n')[0]
+
+            if any(text in desc for text in ['r=merge', 'a=merge']):
```
Comment (jankeromnes) [inline src/shipit_code_coverage/shipit_code_coverage/notifier.py]: Looks like an unrelated drive-by fix, but approved as well.

File: src/shipit_code_coverage/shipit_code_coverage/notifier.py [hunk 2]
```diff
@@ -48,7 +50,7 @@ def notify(self):
                     continue
 
                 if coverage['commit_covered'] < 0.2 * coverage['commit_added']:
-                    content += '* [%s](https://firefox-code-coverage.herokuapp.com/#/changeset/%s): %d covered out of %d added.\n' % (rev, rev, coverage['commit_covered'], coverage['commit_added'])  # noqa
+                    content += '* [%s](https://firefox-code-coverage.herokuapp.com/#/changeset/%s): %d covered out of %d added.\n' % (desc, rev, coverage['commit_covered'], coverage['commit_added'])  # noqa
```
Comment (jankeromnes) [inline src/shipit_code_coverage/shipit_code_coverage/notifier.py]: Interesting. Could you please share a screenshot of what this message looks like out in the wild?

File: src/shipit_code_coverage/shipit_code_coverage/notifier.py (hunk 1 above)
Comment (jankeromnes) [reply src/shipit_code_coverage/shipit_code_coverage/notifier.py]: Ah, nevermind, just realized that you're reusing `desc` below.

File: src/shipit_code_coverage/shipit_code_coverage/notifier.py (hunk 2 above)
Comment (marco-c) [reply src/shipit_code_coverage/shipit_code_coverage/notifier.py]: Here's what it looks like currently:
mail_screen

After this, the hash will be replaced with the first line of the commit message (e.g. `Bug 107264 - Make prefs parser errors louder. r=glandium` in this case).

**Pull request merged**